AlpsSDK/ProximityHandler.swift

```diff
@@ -0,0 +1,233 @@
+//
+//  ProximityHandler.swift
+//  AlpsSDK
+//
+//  Created by Wen on 26.10.17.
+//  Copyright © 2017 Alps. All rights reserved.
+//
+
+import Foundation
+import CoreLocation
+import Alps
+
+class ProximityHandler {
+    
+    var contextManager: ContextManager!
+    static var immediateTrigger: [String: ProximityEvent] = [:]
+    static var nearTrigger: [String: ProximityEvent] = [:]
+    static var farTrigger: [String: ProximityEvent] = [:]
+    static var unknownTrigger: [String: ProximityEvent] = [:]
+    var immediateBeacons: [String] = []
+    var nearBeacons: [String] = []
+    var farBeacons: [String] = []
+    var unknownBeacons: [String] = []
+    var refreshTimer: Int = 60 * 1000 // timer is in milliseconds
+    
+    init(contextManager: ContextManager) {
```

Comment:
Let's make it delegate with some delegate protocol - the same way alps is connected with contextmanager.

It also forces us to define all methods available on delegate.